### ISSUE TITLE
Please update for new-mailboxsearch parameters!

### ISSUE THREAD

**Issue by jreed336** [MicrosoftDocs/OfficeDocs-Exchange#1833]
We need the new formats for these commands!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0e5010fa-a741-a1b1-979d-48e6f61fa5de
* Version Independent ID: b231b1d9-4ff5-d245-f3d2-32cbaecbb66d
* Content: [Clean up or delete items from the Recoverable Items folder](https://docs.microsoft.com/en-us/Exchange/policy-and-compliance/recoverable-items-folder/clean-up-deleted-items?view=exchserver-2019#feedback)
* Content Source: [Exchange/ExchangeServer/policy-and-compliance/recoverable-items-folder/clean-up-deleted-items.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeServer/policy-and-compliance/recoverable-items-folder/clean-up-deleted-items.md)
* Product: **exchange-server-it-pro**
* Technology: ****
* GitHub Login: @mattpennathe3rd
* Microsoft Alias: **v-mapenn**

**Comment by scanum** [comment]
@jreed336  Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by mattpennathe3rd** [comment]
@jreed336 Can you provide a bit more context for this issue? What is missing or broken in the article, please?

**Comment by mhunecke** [comment]
Team, as soon as the search-mailbox command was depreciated, how we can clear the Deliverable Items folder? I had the following cmdlet: 
Search-Mailbox -Identity <Mailbox> -SearchQuery size>0 -SearchDumpsterOnly -DeleteContent -force
The cmdlet that replaces Search-Mailbox would be "New-ComplianceSearchAction", but there is no option to search and delete the Recoverable Items folder (in case -SearchDumpsterOnly parameter).
Thanks

**Comment by mattpennathe3rd** [comment]
@mhunecke Search-Mailbox is not deprecated on Exchange Server (on-premises). This GitHub issue deals with an on-premises environment.

If there is an omission or a problem in an Exchange Online or PowerShell article, please open a new issue on that article. I am not a support representative and I cannot provide support here, aside from helping to correct docs problems.

**Comment by mattpennathe3rd** [comment]
@jreed336 I cannot proceed with this issue unless I have more information about where the problem lies, so I will close this issue now. Please get back in touch if you have any feedback on the documentation.

**Comment by dawilhoit** [comment]
Unless someone has created a way for new-compliancesearch and new-compliancesearchaction to be able to touch the Recoverable Items folder, then I feel like this should be revisited.
search-mailbox was deprecated on 7/1 and theres no way to search content in the recoverable items.
